demes/demes.py

```diff
@@ -909,6 +911,12 @@ def get_demographic_events(self):
         for deme_from, demes_to in splits_to_add.items():
             self.split(deme_from, list(demes_to), self[deme_from].end_time)
 
+    def validate_demography(self):
```

Comment:
Maybe we could call this `validate()`? I think it's clear the demography is going to be validated because it's a method of the DemeGraph.

Reply:
Sounds good! Will change that and merge.